egs2/librimix/diar_enh1/local/prepare_kaldifiles.py

```diff
@@ -0,0 +1,347 @@
+import os
```

Comment:
If it is the same as LibriMix repo, we can do a git clone at `data.sh`. Cause by this way, we can always make sure our version of LibriMix is update to date.

Reply:
It's based on the `s3prl/LibriMix` repo, but I've made some modifications so that it can handle `Libri3Mix` and the `min mode`. I've also modified it so that it outputs `spk{1,2,3}.scp` and `noise1.scp` which are used for separation.

Reply:
Sounds cool!